cypress/integration/experiments.ts

```diff
@@ -0,0 +1,91 @@
+import initialDBData from '../support/initialDBData';
+
+context('Experiments tests', () => {
+  before(() => {
+    cy.resetDB(true);
+  });
+
+  beforeEach(() => {
+    cy.viewport(1920, 1080);
+    cy.resetDB(true);
+    cy.updateProposalManagementDecision({
+      proposalPk: initialDBData.proposal.id,
+      statusId: 1,
```

Comment:
Probably we can use `ProposalEndStatus` enum here instead of just a number for better readability.

Reply:
I think only finalStatus has a datatype of ProposalEndStatus. The statusId is of type number